earthfile2llb/converter.go

```diff
@@ -1143,6 +1181,23 @@ func (c *Converter) Import(ctx context.Context, importStr, as string, isGlobal,
 	return c.varCollection.Imports().Add(importStr, as, isGlobal, currentlyPrivileged, allowPrivilegedFlag)
 }
 
+// Cache handles a `CACHE` command in a Target.
+// It appends run options to the Converter which will mount a cache volume in each successive `RUN` command,
+// and configures the `Converter` to persist the cache in the image at the end of the target.
+func (c *Converter) Cache(ctx context.Context, path string) error {
```

Comment:
Also, should `LOCALLY` do the same?

Reply:
I'd do the same with `LOCALLY` - reset the cache dirs.

Reply:
resolving since other error check covers this